src/keyboard/Keyboard.tsx

```diff
@@ -172,6 +173,15 @@ export default function Keyboard() {
     true
   );
 
+  const [keymapScale, setKeymapScale] = useState<LayoutZoom>(() => {
+    const savedValue = localStorage.getItem("keymapScale");
+    return savedValue === null || savedValue === "auto" ? "auto" : parseFloat(savedValue) || "auto";
+  });
+
+  useEffect(() => {
+    localStorage.setItem("keymapScale", keymapScale.toString());
+  }, [keymapScale]);
+
   const [selectedLayerIndex, setSelectedLayerIndex] = useState<number>(0);
```

Comment:
Given I'm sure we'll have other localStorage state down the road, can we pull this little bit of isolated logic into some new `useLocalStorageState` hook?

Reply:
Done!